IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Types/IfcVirtualElementTypeEnum/DocEnumeration.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IfcVirtualElementTypeEnum" Name="IfcVirtualElementTypeEnum" UniqueId="a28fdf84-8ede-44bd-96ea-b74c66ca5c64" DiagramNumber="1">
+	<DiagramRectangle xsi:type="DocRectangle" X="0" Y="0" Width="400" Height="100" />
+	<Constants>
+		<DocConstant xsi:nil="true" href="BOUNDARY_0D9uazImf8vOPVKzXao_2" />
+		<DocConstant xsi:nil="true" href="CLEARANCE_0hS5MhQVv57hIehqvLOgZ9" />
+		<DocConstant xsi:nil="true" href="PROVISIONFORVOID_2vF_Yr7J92wvfWU_bCEDv" />
```

Comment:
I assume this constant already exists in the repo? (Since it is not marked as added in this PR.)